dashboard/dashboard.py

```diff
@@ -13,762 +11,437 @@
 from email_validator import validate_email as ve, EmailNotValidError
 import phonenumbers
 from phonenumbers import NumberParseException
+from load_dashboard_data import connect_to_db, load_from_starwatch_rds, load_forecasts_by_county_name, load_celestial_body_information
 from nasa_pipeline import nasa_pipeline, get_image_of_the_day, get_iss_location
-from load_dashboard_data import (connect_to_db, load_from_starwatch_rds, load_forecasts_by_county_name,
-                                 load_celestial_body_information)
-
-
 
 load_dotenv()
 st. set_page_config(layout="wide")
```

Comment:
Could put in a blank line between load_dotenv() and the following line but not compulsory.